.devcontainer/base.Dockerfile

```diff
@@ -0,0 +1,55 @@
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 18-bullseye, 16-bullseye, 14-bullseye, 18-buster, 16-buster, 14-buster
+ARG VARIANT=16-bullseye
```

Comment:
Why the second devcontainer `base.Dockerfile`?

Reply:
VSCode put it there automagically